src/analytics/userProperties.ts

```diff
@@ -34,6 +34,8 @@ export interface UserProperties {
   bsc_assets_value?: number;
   deposited_value?: number;
   polygon_assets_value?: number;
+  optimism_assets_value?: number;
+  arbitrum_assets_value?: number;
```

Comment:
imo we should add avalanche & xdai since we are considering adding support and this will give us an insight of if its worth pursuing 


we've also been looking for a way to track nft value for over a year so adding those values as well seems  like another good add